config/webpack.common.js

```diff
@@ -2,17 +2,17 @@
  * @author: @AngularClass
  */
 
-var webpack = require('webpack');
-var helpers = require('./helpers');
+const webpack = require('webpack');
+const helpers = require('./helpers');
```

Comment:
Unfortunately this is deprecated: https://docs.npmjs.com/files/package.json#enginestrict. Please revert that commit.